server/src/test/java/io/crate/integrationtests/ResizeShardsITest.java

```diff
@@ -32,6 +32,7 @@
 import org.elasticsearch.test.IntegTestCase;
 import org.junit.Test;
 
+@IntegTestCase.ClusterScope(numDataNodes = 3, numClientNodes = 0, supportsDedicatedMasters = false)
```

Comment:
Could we keep these randomized?

Reply:
yep, thx! it was for debugging and I forgot, reverted the change.